app/controllers/triannon/auth_controller.rb

```diff
@@ -5,64 +5,56 @@ module Triannon
   class AuthController < ApplicationController
     include RdfResponseFormats
 
-    AUTH_EXPIRY  =   60 # seconds
-    TOKEN_EXPIRY = 3600 # seconds
+    # HTTP request methods accepted by /auth/login
+    LOGIN_ACCEPT = 'OPTIONS, GET, POST'
 
     # OPTIONS /auth/login
     def options
```

Comment:
I *think* OPTIONS is possibly handled well by the rack-cors gem??  But whatever - if it's working now, I'd leave it alone.